pallets/elections/src/lib.rs

```diff
@@ -17,7 +17,6 @@ use sp_std::{
     collections::{btree_map::BTreeMap, btree_set::BTreeSet},
     prelude::*,
 };
-
```

Comment:
Accidental line deletion or is there a reason?

Reply:
Accidental. Restored.